### ISSUE TITLE
publicUrl behind proxy doesn't return static content

### ISSUE THREAD

**Issue by apex-omontgomery** [palantir/policy-bot#259]
I have configured my publicUrl to be 
http://localhost:8080/policy-bot. 
I have configured the github application to use an endpoint https://example.com/policy-bot

This is sitting behind an NGINX proxy and I do not have control over the path based routing scheme. So any request to policybot must be in form of
`https://<server_url>/policy-bot/<suffix>`

This appears to work as desired but the returned static content does not respect the `policy-bot` prefixed path. 

This appears to be due to how the basePath trims everything after trailing slash. 
https://github.com/palantir/policy-bot/blob/develop/server/server.go#L193

I've tried to find ways to rewrite the URL and hostname and every request to 
`https://<server_url>/policy-bot/<suffix>`
returns static content with an href that leads my browser to try to grab the data from
`https://<server_url>/static/....`

```
<!doctype html>
<html lang="en">
    <link rel="icon" href="/static/img/favicon.ico" />
    <link rel="stylesheet" href="/static/css/main.css">
```

What is the recommended way to get around this issue?

**Comment by bluekeyes** [comment]
Sorry about this, we forgot to make the resource paths in the HTML templates dynamic when the base path feature was introduced. It should be fixed by #260.

As a workaround until that merges, you may be able to modify the templates to have the correct paths for your deployment. If running the distribution, the template files are in the `templates` directory and you'll need to modify paths in `page.html.tmpl` and `details.html.tmpl`. If running the Docker image, you could take the templates from a distribution, modify them, and then mount them into the container at the right paths (`/templates/...`) at run time.

**Comment by apex-omontgomery** [comment]
Wow, thank you for the fast response. I'll give that a try.

**Comment by apex-omontgomery** [comment]
I performed a build and verified this works. Do you know when releases are usually cut?

**Comment by bluekeyes** [comment]
We don't have a fixed schedule, so I just released [1.21.1](https://github.com/palantir/policy-bot/releases/tag/v1.21.1) with this fix.
